lib/riemann/tools.rb

```diff
@@ -65,6 +65,7 @@ def report(event)
       end
       
       event = event.merge(attributes)
+      event[:time] = Integer(event[:time]) if event.has_key? :time
```

Comment:
I meant as an alternative to what I'm doing now, not in conjunction with. Basically saying that, if we don't automatically coerce, we should at least inform the developer why the problem might be happening.